schema/examples/direct/multiple_signers/response.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<direct-signature-job-response xmlns="http://signering.posten.no/schema/v1">
+    <signature-job-id>1</signature-job-id>
+    <redirect-url signer="12345678910">
+        https://signering.posten.no#/redirect/P1P-rdPK03Vf-z4NZb1mtDcExjpOzzVEkwal_F9FKm0
+    </redirect-url>
+    <redirect-url signer="10987654321">
+        https://signering.posten.no#/redirect/cwYjoZOX5jOc1BACfTdhuIPjWr4BzZ234PqHOgJj03o
+    </redirect-url>
+    <status-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/status</status-url>
```

Comment:
Bytter ut `{sender-identifier}` med faktisk orgnr. på avsender ("123456789", se examples/direct/multiple_signers/manifest.xml) for å gjøre eksemplene så konkrete som mulig. For klient er dette uansett bare en URL man skal gjøre et kall til, ikke noe man skal "parse" noe ut fra.